addons/project/models/project_task.py

```diff
@@ -1093,12 +1093,7 @@ def _notify_by_email_prepare_rendering_context(self, message, msg_vals=False, mo
             message, msg_vals, model_description=model_description,
             force_email_company=force_email_company, force_email_lang=force_email_lang
         )
-        if self.date_deadline:
-            render_context['subtitles'].append(
-                _('Deadline: %s', self.date_deadline.strftime(get_lang(self.env).date_format)))
-        elif self.date_assign:
-            render_context['subtitles'].append(
-                _('Assigned On: %s', self.date_assign.strftime(get_lang(self.env).date_format)))
+        render_context['subtitles'].append(_('Stage: %s', self.stage_id.name))
```

Comment:
Note that ``stage_id`` is not required, so we could probably avoid displaying 'Stage: False' in emails. Probably remove the stage mention if there is no stage ?

Reply:
@tde-banana-odoo I have made the changes, you can look at it.

Thank you..